meta-balena-common/recipes-containers/resin-supervisor/resin-supervisor/update-resin-supervisor

```diff
@@ -12,12 +12,6 @@ Options:
   -h, --help
         Display this help and exit.
 
-  -i <SUPERVISOR IMAGE>, --supervisor-image <SUPERVISOR IMAGE>
```

Comment:
it might be best to split these changes out as they stand alone i think.

Reply:
I gave it a go but concluded they are not logically separated - using the internal registry means no longer using image and tag, so independent commits do not make sense.